src/components/alert-controller/alert-opts.ts

```diff
@@ -0,0 +1,17 @@
+
+export class AlertOpts {
+
+  /** The dialog title */
+  title?: string;
```

Comment:
I thought it was a reserved word, no warnings when using it ?

Reply:
It's not a component, it's a simple class so no problemo :)